index.d.ts

```diff
@@ -322,7 +332,12 @@ export type IScreenTimeAPI = {
    */
   deleteWebHistoryForURL(url: string, identifier?: string): Promise<void>;
 
-
+  /**
+   * @platform ios
+   * @param {string} token Base64 encoded activity token
+   * @returns {Promise<ActivityLabel>}
+   */
+  getActivityLabel(token: string): Promise<ActivityLabel>;
```

Comment:
there is a chance nothing will be returned if it is a mismatched token or expired token...

```suggestion
  getActivityLabel(token: string): Promise<ActivityLabel | undefined>;
```